common/h/registers/x86_regs.h

```diff
@@ -80,6 +80,7 @@ namespace Dyninst { namespace x86 {
   const signed int YMM    = 0x000B0000;  // YMM0-YMM7 Registers from AVX2/FMA
   const signed int ZMM    = 0x000C0000;  // ZMM0-ZMM7 Registers from AVX-512
   const signed int KMASK  = 0x000D0000;  // K0-K7 opmask Registers from AVX-512
+  const signed int FPCTL  = 0x000E0000;  // 32-bit x87 and SSE control registers
```

Comment:
both 32 and 64-bit and mxcsr is used by avx so change comment to `x87, SSE and AVX control/status registers`.

Reply:
I had only noticed the mention of it in the prose for SSE. Looking at the tables for AVX, it's definitely there. I guess whoever wrote that didn't feel it necessary to explicitly call it out. Good catch.